build.gradle

```diff
@@ -212,7 +212,7 @@ dependencies {
   compile group: 'com.github.hmcts', name: 'java-logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '4.0.0'
 
-  compile group: 'com.github.hmcts', name: 'sscs-common', version: '4.4.45'
+  compile group: 'com.github.hmcts', name: 'sscs-common', version: '4.4.47.RC-SSCS-9403'
```

Comment:
merging RC lib into master

Reply:
can't release it till its tested, will be released when deployed